lighthouse-core/lib/sd-validation/index.js

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
rename `sd-validator.js`(or whatever) so we don't add another `index.js` file :)